### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by sharett**
Hi again Wouter and Bassam - I made some additional commits, and I wasn't sure how to update the old pull request so I made this new one.

### CODE REVIEW

File: database_schema.sql [hunk 1]
```diff
@@ -9,6 +9,19 @@ SET time_zone = "+00:00";
 -- --------------------------------------------------------
 
 --
+-- Table structure for table `blocks`
+--
+
+CREATE TABLE `blocks` (
```
Comment (jakeschievink) [inline database_schema.sql]: I would name this table a different name. I think blocks is too ambiguous and blocked_numbers would convey alot more

File: database_schema.sql [hunk 2]
```diff
@@ -101,25 +114,34 @@ CREATE TABLE `errors` (
 CREATE TABLE `languages` (
   `id` int(11) UNSIGNED NOT NULL,
   `language` varchar(25) NOT NULL,
+  `digit` int(1) UNSIGNED NOT NULL,
```
Comment (jakeschievink) [inline database_schema.sql]: I think digit is too ambiguous as well. Perhaps dial_number? I'm actually having a hard thinking of a good rename all I know is that "digit" hard to understand.

Comment (jakeschievink) [reply database_schema.sql]: maybe menu_number?

File: hotline_blocks.php [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+* @param string &$error
+*   Errors if any occurred.
+* @param string &$message
+*   An informational message if appropriate.
+*   
+* @return bool
+*   True if added successfully.
+*/
+
+function addBlock($phone, &$error, &$message)
```
Comment (jakeschievink) [inline hotline_blocks.php]: Id rename to addBlockedNumber

**Review by jakeschievink — CHANGES_REQUESTED** [review]
Just some naming issues I had. I just gave it a look over, I'll try and give it a thorough test tomorrow.

File: database_schema.sql (hunk 2 above)
Comment (bkurdali) [reply database_schema.sql]: I could change this, let's think : keypress ? digitpressed?

File: lib/lib_sms.php [hunk 4]
```diff
@@ -187,14 +186,20 @@ function sms_getActiveCalls($phone, &$calls, &$error)
...
-
-	_sms_getCallInfo(array("status" => "queued", "to" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "ringing", "to" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "in-progress", "to" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "queued", "from" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "ringing", "from" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "in-progress", "from" => $phone), $client, $calls, $error);
-
+	
+	$statuses = array("queued", "ringing", "in-progress");
```
Comment (bkurdali) [inline lib/lib_sms.php:190]: Is it possible to get the array values from the twilio api? would be DRYer, not a blocker for merging

File: README.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ It is not production ready and is very much in progress.
 
 * Password protect access to the directory you installed it to.  HTTP basic security should work.
 
-* Install a database editing utility such as PhpMyAdmin and place it in the "sanctdb" directory under the main directory.
+* Install a database editing utility such as PhpMyAdmin and place it in a directory under the main directory.
```
Comment (bkurdali) [inline README.md]: Looks like this clobbered a desired change (reverting back to the sanctdb directory) ?

File: lib/lib_sms.php (hunk 4 above)
Comment (sharett) [reply lib/lib_sms.php:190]: Sorry I don't really understand.  These are the status values that we want to retrieve from the Twilio API.   There are more status values, but they are for completed calls.

File: README.md (hunk 5 above)
Comment (sharett) [reply README.md]: No, I changed it back.  I realized that it needs to be called "sanctdb" for now as that is the name in .gitignore

File: database_schema.sql (hunk 2 above)
Comment (sharett) [reply database_schema.sql]: How about menu_selection?

**Pull request merged**